no4/index.html

```diff
@@ -0,0 +1,153 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>web to do mate</title>
+    <script
+      src="https://kit.fontawesome.com/bef409a910.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <main>
+      <header>
+        <h2>🤍 WEB TO DO MATE 🤍</h2>
+      </header>
+      <section id="calender">
+        <article id="one_day">
+          <div id="day">Mon</div>
+          <div id="icon">
+            <i class="fa-solid fa-clover"></i>
+            <div id="icon_no">4</div>
+          </div>
+          <div id="date">3</div>
+        </article>
+
+        <article id="one_day">
+          <div id="day">Tue</div>
+          <div id="icon">
+            <i class="fa-solid fa-clover"></i>
+            <div id="icon_no">5</div>
+          </div>
+          <div id="date">4</div>
+        </article>
+
+        <article id="one_day">
+          <div id="day">Wed</div>
+          <div id="icon">
+            <i class="fa-solid fa-clover"></i>
+            <div id="icon_no">3</div>
+          </div>
+          <div id="date">5</div>
+        </article>
+
+        <article id="one_day">
+          <div id="day">Thu</div>
+          <div id="icon">
+            <i class="fa-solid fa-clover"></i>
+            <div id="icon_no">6</div>
+          </div>
+          <div id="date">6</div>
+        </article>
+
+        <article id="one_day">
+          <div id="day">Fri</div>
+          <div id="icon_selected">
+            <i class="fa-solid fa-clover"></i>
+            <div id="icon_no_selected">4</div>
+          </div>
+          <div id="date_selected">7</div>
+        </article>
+
+        <article id="one_day">
+          <div id="day">Sat</div>
+          <div id="icon">
+            <i class="fa-solid fa-clover"></i>
+            <div id="icon_no">2</div>
+          </div>
+          <div id="date">8</div>
+        </article>
+
+        <article id="one_day">
+          <div id="day">Sun</div>
+          <div id="icon">
+            <i class="fa-solid fa-clover"></i>
+            <div id="icon_no">2</div>
+          </div>
+          <div id="date">9</div>
+        </article>
+      </section>
+      <section id="list">
+        <article id="todo">
+          <div id="everyday">
+            <h3 class="everyday">
+              Everyday
+              <i class="fa-solid fa-circle-plus"></i>
+            </h3>
```

Comment:
> todo list 안의 icon 에도 똑같은 방법으로 텍스트를 입히려 했더니 안되더라구요…? 정말 왜 안되는걸까….???? 일단 아이콘에 자체적으로 플러스 사인이 있는 아이콘으로 대체해서 완성했습니다..
```suggestion
            <h3 class="everyday">
              Everyday
              <div class="icon_wrapper">
                <i class="fa-solid fa-circle-plus"></i>
                <p class="icon_content">+</p>
              </div>
            </h3>
```
요기도 요렇게 아이콘이랑 글자를 한번 더 감싸준 다음에

``` css
.icon_wrapper {
  position: relative;
}

.icon_content {
  position: absolute;
  top: 50%;
  left: 50%;
  transform: translate(-50%, -50%);
}
```
달력부분처럼 똑같이 이렇게 속성 적용시켜 주면 중간에 위치시킬 수 있습니당!!!ㅎㅎㅎ

Reply:
하 드디어 알았다..🥹 고마워요오